libraries/anaconda3-buildtime/context/build

```diff
@@ -4,6 +4,11 @@ set -e
 
 ROOTDIR=/opt/algorithm
 
```

Comment:
Can we add a comment for what $1 is supposed to be

Reply:
yea that's not a bad idea, I'll add one